src/app/components_ngrx/group-types-panel/group-types-panel.component.ts

```diff
@@ -66,7 +68,11 @@ export class GroupTypesComponent implements OnInit, OnDestroy {
         if (!this.startedCheckingURL) {
           this.checkURL();
         }
-      })
+      }),
+      this.store
+        .select('listPage')
+        .select('infotips')
+        .subscribe(i => this.infotips = i)
```

Comment:
We planned not to subscribe to observables from the component, use async pipe instead. What happened to that?